docs/src/manual.md

```diff
@@ -309,3 +308,203 @@ The resulting dual model is
     && con & \in FakeDualCone(3)\\
 \end{align}
 ```
+
+## Advanced
+
+### KKT Conditions
+
+The KKT conditions are a set of inequalities for which the feasible solution is equivalent to the optimal solution of an optimization problem, as long as strong duality holds and constraint qualification rules such as Slater's are valid. The KKT is used in many branches of optimization and it might be interesting to write them programatically.
+
+The KKT conditions of the minimization problem of the first section are the following:
+
+1. Primal Feasibility:
+
+```math
+A_i x + b_i  \in \mathcal{C}_i , \ \ i = 1 \ldots m
+```
+
+2. Dual Feasibility:
+
+```math
+y_i \in \mathcal{C}_i^*, \ \ i = 1 \ldots m
+```
+
+3. Complementary slackness:
+
+```math
+y_i^T (A_i x + b_i) = 0, \ \ i = 1 \ldots m
```

Comment:
The choice of sense here is pretty important for solvers such as PATH; see the "Pitfalls" section of https://www.gams.com/latest/docs/S_PATH.html.

I think this is the correct way, but it might be worth adding a note (and maybe tests?) so that we don't change the definition in future.

Reply:
From what I could understand path has complementarity for all constraints, while I just wrote for primal constraint + dual variables, I did not write anything for dual constraints + primal variables

Reply:
You could re-write the complementarity constraint with a `-` in front of the primal constraint: `y_i^T (-A_i x - b_i) = 0`. 

Given our conic notation, your way is the obvious thing to do. It's more a problem with `Ax = b`, when you could formulate `Ax - b = 0` or `b - Ax = 0`. We want `Ax - b`.

Reply:
I see! then we are just all right.
About the KKT itself, I might leave the code for another PR.
It works in BilevelJuMP.jl for now.